packages/design/src/components/form/custom-file-field.tsx

```
@@ -0,0 +1,418 @@
+                      {field.value ? "New file selected" : "Current file"}
+                    </p>
+                  </div>
+                ) : field.value ? (
+                  <div className="flex items-center space-x-2">
+                      {/* eslint-disable-next-line @typescript-eslint/no-unsafe-argument */}
+                    {getFileIcon(field.value.type ?? "")}
+                    <span className="flex flex-col gap-1 text-sm text-xs font-medium">
+                          {/*eslint-disable-next-line @typescript-eslint/no-unnecessary-condition*/}
+                      {truncateFileName(field.value.name as string, 20) ?? (typeof field.value === "object"
+                        ? Object.entries(field.value).map(([_, file]) => (
+                            <span>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)